docs/ref/templates/builtins.txt

```diff
@@ -2430,18 +2430,16 @@ slightly different call:
     {% static "images/hi.jpg" as myphoto %}
     <img src="{{ myphoto }}"></img>
 
-.. note::
 
-    The :mod:`staticfiles<django.contrib.staticfiles>` contrib app also ships
-    with a :ttag:`static template tag<staticfiles-static>` which uses
-    ``staticfiles'`` :setting:`STATICFILES_STORAGE` to build the URL of the
-    given path (rather than simply using :func:`urllib.parse.urljoin` with the
-    :setting:`STATIC_URL` setting and the given path). Use that instead if you
-    have an advanced use case such as :ref:`using a cloud service to serve
-    static files<staticfiles-from-cdn>`::
+If :mod:`staticfiles<django.contrib.staticfiles>` contrib app is configured,
+the static template tag, will serve all files from
+defined storage in :setting:`STATICFILES_STORAGE`.
+
+.. versionchanged:: 1.10
 
-        {% load static from staticfiles %}
-        <img src="{% static "images/hi.jpg" %}" alt="Hi!" />
+    In the past static had to be loaded form :mod:`staticfiles<django.contrib.staticfiles>`
```

Comment:
In older versions, ...
add backticks around static
~django.contrib.staticfiles